pages/manual/manual-gettingstarted-build.md

```diff
@@ -100,22 +110,56 @@ Unless you specifically need to debug the application, we recommend setting the
 
 
 ### Mac
-TODO: do
-{% include note.html content="
-When using a multi-configuration generator (like Xcode) you may want to adjust your build mode manually, since it probably defaults to `Debug` at first, which has a large impact on the performance.
-
-If you use a single-configuration generator, you can control the build mode using `CMAKE_BUILD_TYPE` in CMake.
+#### Dependencies
+You will need at least
+- [XCode](https://developer.apple.com/xcode/) 
+- [CMake](https://cmake.org/download/) >= 3.12.0
+- [Qt binaries](https://qt.io/download-open-source/) >= 5.12 
```

Comment:
same as above, but no less then 5.15 and prefer 6+

Reply:
6+ is not well tested yet as I have not thouroughly tested packaging

Reply:
no one more then you and me some years ago uses that. (maybe Martin, be he will manage) So I don't think that is so important... 5 will not work on arm for example...